scripts/dockerfiles/Dockerfile.init-release

```diff
@@ -9,5 +9,9 @@ COPY --from=init-builder /fluent_bit_init_process /init/fluent_bit_init_process
 COPY init/fluent_bit_init_entrypoint.sh /init/fluent_bit_init_entrypoint.sh
 RUN chmod +x /init/fluent_bit_init_entrypoint.sh
 
+# Change to non-root user
+RUN groupadd -r user && useradd -r -g user user
+USER user
+
```

Comment:
We can't do this by default. In FireLens, we need to run FLB as root. I've been meaning to create an FAQ on this for some time. I'll send you an internal note for it.

Reply:
Can we create the user and then optionally customers can run FLB as it at runtime if they want?

Reply:
I know k8s has a RunAsUser field. I'm not experienced with it though.